libs/@local/repo-chores/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@local/repo-chores",
+  "version": "0.0.0-private",
+  "private": true,
+  "scripts": {
+    "exe": "ts-node --transpile-only",
+    "fix:eslint": "eslint --fix .",
+    "lint:eslint": "eslint --report-unused-disable-directives .",
+    "lint:tsc": "tsc --noEmit"
+  },
+  "dependencies": {
+    "chalk": "4.1.2",
+    "execa": "5.1.1",
+    "fs-extra": "10.1.0",
+    "globby": "11.0.1",
+    "typescript": "4.9.4"
```

Comment:
`devDependencies`?

Reply:
We have
```
    "ts-node": "10.9.1",
    "typescript": "4.9.4"
```
in other workspaces in which we have `yarn exe`. I added `ts-node` in https://github.com/hashintel/hash/pull/1825/commits/507d472eca160d5ecf5154f7b48516bf9ce19e1d. If that’s inaccurate, we can move these dependencies in all workspaces one day, but at least it will be consistent for now.